numpy/core/_internal.py

```diff
@@ -288,22 +288,32 @@ def _newnames(datatype, order):
     raise ValueError("unsupported order value: %s" % (order,))
 
 # Given an array with fields and a sequence of field names
-# construct a new array with just those fields copied over
-def _index_fields(ary, fields):
-    from .multiarray import empty, dtype, array
+# construct new array with just those fields
+def _index_fields(ary, names):
+    from .multiarray import dtype, array
```

Comment:
Just noticed, will merge without, but if you are bored and want to increase your commit count.... We should probably move all these imports here to the module level.